schema/me/index.js

```diff
@@ -31,6 +32,9 @@ const Me = new GraphQLObjectType({
     paddle_number: {
       type: GraphQLString,
     },
+    disqualified: {
```

Comment:
Where is this field coming from? If we want to see whether a user is disqualified from bidding, I think we'll actually want to check the `qualified_for_bidding` field returned in the user's `Bidder` json for a sale which also checks for whether an admin has explicitly approved them.